Containers/fulltextsearch/Dockerfile

```diff
@@ -0,0 +1,4 @@
+# Probably from here https://github.com/elastic/elasticsearch/blob/main/distribution/docker/src/docker/Dockerfile
+FROM elasticsearch:7.17.5
+
+RUN elasticsearch /usr/share/elasticsearch/bin/elasticsearch-plugin install --batch ingest-attachment
```

Comment:
I think this does not build
![grafik](https://user-images.githubusercontent.com/75573284/186651507-7c93b8c6-3db3-4d16-b02d-2614926f0371.png)

If I replace the RUN command with: `elasticsearch-plugin install --batch ingest-attachment`
It outputs:
![grafik](https://user-images.githubusercontent.com/75573284/186651674-bb30f720-0d0a-4374-9c97-7cc27a28ad66.png)

Reply:
you are right. Fixed with https://github.com/nextcloud/all-in-one/commit/6c63adc37ff0f755375cea2a518bb0adc11560f4